### PR TITLE
python-imglib/pillow: replace discontinued python-imglib (PIL) with pillow

### PR DESCRIPTION
**Pull request by kissg1988**
PIL's development appears to be discontinued with the last commit to the PIL repository coming in 2011. This change replaces PIL with Pillow, a fork of PIL still regularly updated.

Also, there is a major vulnerability for the latest version of PIL (CVE-2014-3007) not likely to be fixed.

### CODE REVIEW

**Comment by jow-** [comment]
Are there any users of python-pil in the feed which we need to consider?

**Comment by kissg1988** [comment]
Only seafile-server uses it as far as I know (it was imported as a dependency of seafile-server earlier).

File: lang/pillow/patches/010-fix-paths.patch [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+ 
+ 
+@@ -187,129 +187,6 @@ class pil_build_ext(build_ext):
+             _add_directory(library_dirs, os.path.join(prefix, "lib"))
+             _add_directory(include_dirs, os.path.join(prefix, "include"))
+ 
+-        #
+-        # add platform directories
+-
+-        if sys.platform == "cygwin":
```
Comment (jow-) [inline lang/pillow/patches/010-fix-paths.patch]: To simplify the patch it maybe makes sense to add a dummy `if` clause here, something like:

    if os.environ['STAGING_DIR']:
        print("Not altering search path")
    elif sys.platform == "cygwin":
        ...

**Comment by jow-** [comment]
I compile tested pillow, worked here, merged.

**Pull request merged**